src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/JdbcTestIT.java

```diff
@@ -137,4 +137,32 @@ public void ipTypeShouldPassJdbcFormatter() {
                 containsString("\"type\": \"ip\"")
         );
     }
+
+    @Test
```

Comment:
is this case-sensitive? If my query contains "SELECT SUBSTRING(lastname, 1, 2)" will it return "substring" or "SUBSTRING"

Reply:
Yep it is. it returns SUBSTRING